app/frontend/src/features/messages/requests/HostRequestListItem.tsx

```diff
@@ -105,7 +110,7 @@ export default function HostRequestListItem({
                 true
               )}`}
             </Typography>
-            <TextBody noWrap>
+            <TextBody noWrap className={isUnreadClass}>
```

Comment:
We use the `classNames` package for dealing with this kind of logic, so you can do something like this instead:

```suggestion
            <TextBody noWrap className={classNames({ [classes.unread]: isUnread })} />
```

Reply:
This makes sense, thanks for letting me know. I made the change in the latest commit.